src/applications/mhv/landing-page/components/NoHealthAlert.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const NoHealthAlert = () => {
+  return (
+    <va-alert
+      close-btn-aria-label="Close notification"
+      status="info"
+      visible
+      data-testid="no-health-message"
+    >
+      <h2 slot="headline">We cannot display your health information.</h2>
+      <div>
+        <p className="vads-u-margin-y--0">
+          Only patients who have received care at a VA facility can use VA.gov
+          health tools. If you’ve received care at a VA medical center, clinic,
+          or Vet center, please call that facility to find out if you’re in
+          their records.
```

Comment:
Looks like the text content may have changed: https://app.zenhub.com/workspaces/mhv-on-vagov-top-level-view-62619a987d74510018ecc546/issues/gh/department-of-veterans-affairs/va.gov-team/67089#issuecomment-1773049473

Reply:
Done, but it may change again.